PowerEditor/src/NppBigSwitch.cpp

```diff
@@ -219,7 +219,8 @@ LRESULT Notepad_plus::process(HWND hwnd, UINT message, WPARAM wParam, LPARAM lPa
 
 		case WM_FINDALL_INCURRENTDOC:
 		{
-			return findInCurrentFile();
+			bool searchScopeIsSelectedText = (wParam == 0) ? false : true;

```

Comment:
Perhaps, **`bool searchScopeIsSelectedText = wParam != 0`**?

Reply:
Yea, my code is strange there.
I've got to believe it came from an evolution of a slightly different bit of logic I was working on there, but I don't have any way to remember that. :-)